css-view-transitions-1/Overview.bs

```diff
@@ -1020,27 +1045,24 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 		1. Let |document| be [=this's=] [=relevant global object's=] [=associated document=].
 
-		1. Let |transitionRoot| be the result of creating a new ''::view-transition'' pseudo-element in |document|'s [=snapshot viewport=].
+		1. Set |document|'s [=show view-transition root pseudo-element=] to true.
```

Comment:
That's it, why the change from generating the root here to assuming it always exists? I found lifetimes easier to follow when this element was generated here (with rest of the pseudos) and torn down in clear view transition.

Reply:
Ok. I'm fine with this pattern too.